homeassistant/components/ledsc/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "ledsc",
+  "name": "LedSC",
+  "codeowners": ["@PatrikKr010"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/ledsc/",
+  "integration_type": "hub",
```

Comment:
```suggestion
  "integration_type": "device",
```

Reply:
But device is not the right category. It is really a hub. The integration is connected to a single device using Websocket, but this device controls any number of lights.

The individual devices are configured in the WebSC application. Advanced LedSC properties can be configured in this application. The integration connects to WebSC and loads the configured devices. So it is not necessary to configure the individual devices separately in the integration configuration.

Reply:
you are right